api/README.md

```diff
@@ -2,24 +2,19 @@
 
 The lightweight API is a stripped-down alternative to the WordPress REST API. Prompts will often be present on every page of a site, which can lead to performance issues as a result of a large number of uncacheable API requests.
 
-The feature requires a custom config file named `newspack-popups-config.php` at the root of the site, which will be created at plugin initialisation.
+## Configuration
 
-To include information about database, cache and time in the response, add this constant to that config file:
+The feature requires a custom config file named `newspack-popups-config.php` at the root of the site. See the main README for more information.
 
-```
-define( 'NEWSPACK_POPUPS_DEBUG', true );
-```
+## API Overview
 
-## Manually creating the config file
+| Endpoint | Frequency | Purpose |
+| :------------- | :------------- | :------------- |
+| `GET /api/campaigns/index.php` | Page load | Tells the page which prompts should be displayed. This is the [authorization endpoint](https://amp.dev/documentation/components/amp-access/#authorization-endpoint) of the `amp-access` AMP component. |
```

Comment:
This is something I only recently understood, but it would be nice to reference how the endpoint knows which campaigns to check for authorization. I understand that this happens [here](https://github.com/Automattic/newspack-popups/blob/master/includes/class-newspack-popups-inserter.php#L410), and that this function runs relatively independently of which prompts are actually rendered on the page (because some prompts may be in post content, others in widgets or shortcodes).

Reply:
Added a column with references to the methods - would that suffice?